src/App.tsx

```diff
@@ -1,24 +1,146 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+/* eslint-disable no-console */
+import React, { useEffect, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Eror } from './components/ERROR';
+import { getTodos, addTodo, deleteTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { FilterBy } from './types/FilterEnum';
 
-const USER_ID = 0;
+const USER_ID = 10387;
+
+export enum Error {
+  LOAD = 'load',
+}
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [title, setTitle] = useState<string>('');
+  const [isDeleting, setIsDeleting] = useState(false);
+
+  const visibleTodos = todos.filter((todo) => {
+    switch (filterBy) {
+      case FilterBy.Active:
+        return !todo.completed;
+
+      case FilterBy.Completed:
+        return todo.completed;
+
+      default:
+        return true;
+    }
+  });
+
+  const addError = (error: Error) => {
+    setErrorMessage(error);
+    window.setTimeout(() => {
+      setErrorMessage(null);
+    }, 3000);
+  };
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      try {
+        setErrorMessage(null);
+        const todosFromServer = await getTodos(USER_ID);
+
+        setTodos(todosFromServer);
+      } catch (error) {
+        addError(Error.LOAD);
+      }
+    };
+
+    loadTodos();
+  }, []);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!title) {
+      setErrorMessage('Title cannot be empty');
+
+      return;
+    }
+
+    const newTodo = {
+      id: 0,
+      userId: USER_ID,
+      title,
+      completed: false,
+    };
+
+    setTempTodo(newTodo);
+
+    if (tempTodo) {
+      addTodo(tempTodo)
+        .then((response) => {
+          setTodos([...todos, response]);
+          setTitle('');
+        })
+        .catch(() => setErrorMessage('Unable to add a todo'))
+        .finally(() => {
+          setTempTodo(null);
+        });
+    }
+  };
+
+  const handleDelete = (id: number) => {
+    setIsDeleting(true);
+
+    deleteTodo(id)
+      .then(() => {
+        setTodos(todos.filter((todo) => todo.id !== id));
+      })
+      .catch(() => setErrorMessage('Unable to delete a todo'))
+      .finally(() => setIsDeleting(false));
+  };
+
+  console.log(todos);
```

Comment:
Remove the 'console.log(todos);' statement as it is not necessary and may expose sensitive information.